bitwarden_keyring/__init__.py

```diff
@@ -174,8 +174,9 @@ def get_session(environ):
     return ask_for_session(bool(user))
 
 
-def get_password(service, username):
-    session = get_session(os.environ)
+def get_password(service, username, session=None):
+    if session == None:
```

Comment:
```suggestion
    if session is None:
```

Reply:
What about passing session to `get_session()` and doing the check over there ? This year, we won't have the check in multiple places in the code ?